collectington/config.py

```diff
@@ -85,9 +84,6 @@ def validate(config):
             "Invalid config: config should contain port, api_call_intervals, log_level, services"
         )
 
-    if not isinstance(config["port"], int):
-        raise ValueError("Invalid config: port should be an integer")
-
```

Comment:
**Change**: can you add this test to the `validate_service` section so we're still validating the port has the correct data type?

Reply:
done!